src/NativeBridge/DataViewInterop.h

```diff
@@ -243,15 +243,23 @@ class DataSourceBlock
             size = -1;
             missing = -1;
             pch = bp::extract<const char*>(str(s).encode("utf_8"));
+            if (str(s).encode("utf_8").is_none())
+            {
+                size = 0;
```

Comment:
>size = 0; [](start = 16, length = 9)

When do we come here?

Reply:
For context, the `is_none `function and its concerning values:

```
inline bool api::object_base::is_none() const
{
    return (m_ptr == Py_None);
}
```
`PyObject* m_ptr;
`
```
/*
_Py_NoneStruct is an object of undefined type which can be used in contexts
where NULL (nil) is not suitable (since NULL often means 'error').

Don't forget to apply Py_INCREF() when returning this value!!!
*/
PyAPI_DATA(PyObject) _Py_NoneStruct; /* Don't use this directly */
#define Py_None (&_Py_NoneStruct)
```
I believe the code reaches in this if statement in line 246 if the given object is equal to the `NullType `in Python.
The peculiar question is, on Linux builds this statement works fine, but in our Mac builds, this statement doesn't fix the failure. Perhaps `Py_None` doesn't exactly point to  `NoneType `on Mac? In addition, this test was being skipped until PR #359, so this means this issue did not surface until this test was no longer being skipped.